[PL-41558] Update copy token info, add permission requirement and screenshot (#3509)
harness/developer-hub: docs/platform/delegates/secure-delegates/secure-delegates-with-tokens.md

   ![](static/secure-delegates-with-tokens-03.png)

> 2. Select **Tokens**. Here you can view, create, and revoke all delegate tokens. You can also use the **More Options** (&vellip;) menu to view **More info** or **Copy token**. You must have the Create Delegate permission to copy a token.

Reviewer: ```suggestion
2. Select **Tokens**. Here you can view, create, and revoke all delegate tokens. For each token, you can select More Options (**&vellip;**) menu to view **More info** or **Copy token**. You must have the Create Delegate permission to copy a token.
```
I'm thinking it might make more sense to bold the dot-dot-dot rather than More Options (does the label appear when you hover over the dot-dot-dot?) but this is probably a subjective preference...

Reviewer: https://developers.google.com/style/word-list#hamburger
